spring-integration-file/src/main/java/org/springframework/integration/file/tail/OSDelegatingFileTailingMessageProducer.java

```diff
@@ -71,13 +72,13 @@ public boolean isLongLived() {
 	protected void onInit() {
 		Assert.notNull(getFile(), "File cannot be null");
 		super.onInit();
-		this.command = "tail " + this.options + " " + this.getFile().getAbsolutePath();
 	}
 
 	@Override
 	protected void doStart() {
 		super.doStart();
 		destroyProcess();
+        this.command = "tail " + this.options + " " + this.getFile().getAbsolutePath();
```

Comment:
Hi @bitwisekaizen !
Everything looks good. Just this minor issue: be sure that you use `tabs` for indents, not `whitespaces`.
Can be address on merge.

We need the final approval from @garyrussell, as from an original author, and merge.

Thank you very much for the contribution. Looking forward for more!